src/commands/pull.js

```diff
@@ -0,0 +1,150 @@
+const fs = require('fs');
+const path = require('path');
+const tiged = require('tiged');
+const AdmZip = require('adm-zip');
+const semverGt = require('semver/functions/gt');
+const { get } = require('axios');
+const { flags: flagsHelper } = require('@oclif/command');
+const json5 = require('json5');
+const { ApifyCommand } = require('../lib/apify_command');
+const { success, error } = require('../lib/outputs');
+const { getLoggedClientOrThrow, getLocalConfigOrThrow, getLocalUserInfo } = require('../lib/utils');
+const { LOCAL_CONFIG_PATH } = require('../lib/consts');
+
+const extractGitHubZip = async (url, directoryPath) => {
+    const { data } = await get(url, { responseType: 'arraybuffer' });
+
+    const zipFile = new AdmZip(Buffer.from(data, 'binary'));
+
+    zipFile.extractEntryTo(zipFile.getEntries()[0].entryName, directoryPath, false);
+};
+
+class PullCommand extends ApifyCommand {
+    async run() {
+        const { args, flags } = this.parse(PullCommand);
+        const localConfig = await getLocalConfigOrThrow();
+        const userInfo = await getLocalUserInfo();
+        const apifyClient = await getLoggedClientOrThrow();
+        const cwd = process.cwd();
+
+        const isActorAutomaticallyDetected = !args?.actorId;
+        const usernameOrId = userInfo.username || userInfo.id;
+
+        const actorId = args?.actorId || localConfig?.id || (localConfig.name ? `${usernameOrId}/${localConfig.name}` : undefined);
+
+        if (!actorId) throw new Error('Cannot find actor in this directory.');
+
+        const actor = await apifyClient.actor(actorId).get();
```

Comment:
Let's wrap this with a try-catch and some nice error message, otherwise you can get seemingly irrelevant errors:
<img width="665" alt="Screenshot 2023-05-29 at 17 54 20" src="https://github.com/apify/apify-cli/assets/2934111/698c4b87-91a9-470f-8090-f820ac0b12d3">